tutorials/inverse/95_phantom_KIT.py

```diff
@@ -0,0 +1,21 @@
+"""
+.. _tut-phantom-kit:
+
+============================
+KIT phantom dataset tutorial
+============================
+
+Here we read KIT data TODO EXPLAIN.
+"""
+
+# Authors: Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD-3-Clause
+
+# %%
+import mne
+
+data_path = mne.datasets.phantom_kit.data_path()
+# TODO: This should *not* need the ".." but it does b/c of how the `.zip` was created
+raw = mne.io.read_raw_kit(data_path / ".." / "002_phantom_11Hz_100uA.con")
```

Comment:
When I actually run this functional locally it unzips incorrectly:
```Console
$ python -c "import mne; print(mne.datasets.phantom_kit.data_path(verbose=True))"
Downloading file 'MNE-phantom-KIT-24bit.zip' from 'https://osf.io/download/ed3j8/' to '/home/larsoner/mne_data'.
100%|█████████████████████████████████████| 1.01G/1.01G [00:00<00:00, 5.83TB/s]
Unzipping contents of '/home/larsoner/mne_data/MNE-phantom-KIT-24bit.zip' to '/home/larsoner/mne_data'
Download complete in 01m03s (966.2 MB)
/home/larsoner/mne_data/MNE-phantom-KIT-data
$ ls /home/larsoner/mne_data/MNE-phantom-KIT-data
ls: cannot access '/home/larsoner/mne_data/MNE-phantom-KIT-data': No such file or directory
$ $ ls -alt /home/larsoner/mne_data/
total 2910108
drwxr-xr-x  68 larsoner larsoner       4096 Oct 17 15:23 .
-rw-rw-r--   1 larsoner larsoner    2891472 Oct 17 15:23 230831-ini.mrk
-rw-rw-r--   1 larsoner larsoner    2891472 Oct 17 15:23 230831-final.mrk
-rw-rw-r--   1 larsoner larsoner 1397822672 Oct 17 15:23 002_phantom_11Hz_100uA.con
drwxr-xr-x 188 larsoner larsoner      12288 Oct 17 15:20 ..
drwxrwxr-x  36 larsoner larsoner       4096 Oct 10 09:37 MNE-testing-data
...
```
Is it possible your `.zip` just contains these three files? If so, could you add them to a `MNE-phantom-KIT-data` folder, zip that folder, and re-upload it (and update the `?version=` and hash)? Then I think it should extract properly, and the `".."` should be removable here.

You can test locally with `python -c "import mne; mne.datasets.phantom_kit.data_path(force_download=True, verbose=True)"` then check to make sure the `.con` etc. files land in `~/mne_data/MNE-phantom-KIT-data` rather than `~/mne_data`

Reply:
Thank you and sorry for the delay! I have now re-uploaded the files in the correct format, and updated the URL and hash in config.py
I couldn't test it out locally though as I kept hitting an issue with the lazy_loader module.